pkg/kubelet/dockertools/manager.go

```diff
@@ -47,6 +47,7 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/prober"
 	"k8s.io/kubernetes/pkg/kubelet/qos"
 	kubeletTypes "k8s.io/kubernetes/pkg/kubelet/types"
+	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
```

Comment:
if it is this way in head, we can go ahead with this but we should fix head ASAP